src/components/Button.js

```diff
@@ -1,14 +1,32 @@
 import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import PropTypes from 'prop-types';
+import './Button.css';
 
 function Button(props) {
-  const { name } = props;
-  return <span>{name}</span>;
+  const { name, wide, color } = props;
+
+  const wideStyle = {
+    width: '49.7%',
+    backgroundColor: color,
+  };
+
+  const normalStyle = {
+    width: '24.7%',
+    backgroundColor: color,
+  };
```

Comment:
We should use `25%` for all buttons and `50%` for the `0` button in here.
![image](https://user-images.githubusercontent.com/31897434/94064349-76c03b80-fde1-11ea-97ec-8872f127e142.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896093/